app-i18n/kime/kime-2.5.0.ebuild

```diff
@@ -301,7 +301,6 @@ src_install() {
 	insopts -m0644
 	insinto /etc/xdg/kime
```

Comment:
Since 2.5.0 kime doesn't install config file please follow this install script https://github.com/Riey/kime/blob/develop/scripts/install.sh